cluster/images/hyperkube/Makefile

```diff
@@ -1,6 +1,9 @@
-# build the hyperkube image.
+# Build the hyperkube image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-VERSION=v1.1.2
+REGISTRY?="gcr.io/google_containers"
```

Comment:
I'm fine with removing default versions to prevent that it out-dates fast.
I just wanted to make sure it wasn't by mistake or something.